packages/field-base/src/styles/field-shared-styles.js

```diff
@@ -23,20 +24,52 @@ export const fieldShared = css`
     display: none !important;
   }
 
-  :host(:not([has-label])) [part='label'] {
+  :host(:not([has-label])) [part='label'],
+  :host(:not([has-helper])) [part='helper-text'],
+  :host(:not([has-error-message])) [part='error-message'] {
     display: none;
   }
 
-  @media (forced-colors: active) {
-    :host(:not([readonly])) [part='input-field'] {
-      outline: 1px solid;
-      outline-offset: -1px;
-    }
-    :host([focused]) [part='input-field'] {
-      outline-width: 2px;
-    }
-    :host([disabled]) [part='input-field'] {
-      outline-color: GrayText;
-    }
+  [part='label'] {
+    font-size: var(--vaadin-input-field-label-font-size, inherit);
+    line-height: var(--vaadin-input-field-label-line-height, inherit);
+    font-weight: var(--vaadin-input-field-label-font-weight, 500);
+    color: var(--vaadin-input-field-label-color, var(--_vaadin-color-strong));
+    order: var(--vaadin-input-field-helper-order);
+  }
+
+  :host([required]) [part='required-indicator']::after {
+    content: var(--vaadin-input-field-required-indicator, '*');
```

Comment:
I would suggest to also include / migrate `--lumo-required-field-indicator-color` to the base styles - e.g. as `--vaadin-input-field-required-indicator-color`

Reply:
Right, good catch.